### PR TITLE
Built featured-projects list.

### PR DESCRIPTION
**Pull request by KyleSaunders1310**

### CODE REVIEW

File: src/app/components/projects-page/projects-page.component.scss [hunk 1]
```diff
@@ -6,6 +6,111 @@
...
+  display: flex;
+  margin: 20px 0px;
+  &:hover img {
+    filter: grayscale(0);
+    opacity: 100%;
+  }
+  &:nth-child(odd){
+    flex-direction: row;
+    left: 0;
+    & .featured-project-info {
```
Comment (Solenoden) [inline src/app/components/projects-page/projects-page.component.scss:32]: The & is redundant

File: src/app/components/projects-page/projects-page.component.scss [hunk 2]
```diff
@@ -6,6 +6,111 @@
...
+
+carousel {
+  height: 100% !important;
+  width: 100%;
+  border-radius: 10px;
+  z-index: -1;
+  filter: grayscale(100%);
+  opacity: 40%;
+
+  & img {
```
Comment (Solenoden) [inline src/app/components/projects-page/projects-page.component.scss:97]: Quite a few & in this file are redundant. A nested selector is implicitly the child of the parent selector

**Review by Solenoden — CHANGES_REQUESTED** [review]

File: src/app/components/projects-page/projects-page.component.scss (hunk 1 above)
Comment (KyleSaunders1310) [reply src/app/components/projects-page/projects-page.component.scss]: Fixed

File: src/app/components/projects-page/projects-page.component.scss (hunk 2 above)
Comment (KyleSaunders1310) [reply src/app/components/projects-page/projects-page.component.scss]: Fixed

**Review by Solenoden — APPROVED** [review]

**Pull request merged**